src/App/screens/Instructor/state/reducers/index.js

```diff
@@ -1,7 +1,7 @@
 import {combineReducers} from 'redux';
 import * as instructorActionTypes from '../actions/instructorActionTypes';
 
-const instructor = (state = false, action) => {
+const instructor = (state = {firstLessonState: 'approved'}, action) => {
```

Comment:
I don't even know that it needs to be in the store. Is it shared? Maybe it's just component local? Server won't really know about any of this though so that is up to you.